### PR TITLE
updates auction, vaults

### PR DESCRIPTION
**Pull request by DongLieu**
This was updated from the dev-oraclemock branch after testing.

App: use app v1

Auction:
- querry auction
- remove startTime,  endTime
- use lastestAuctionPeriod primitive type int64 instead collections
- updates abci

Vaults:
- liquidate
- gov
- param DefaultMinInitialDebt 20_000_000 up to 50_000_000
- remove param LiquidatePeriod

### CODE REVIEW

File: x/vaults/keeper/vault.go [hunk 1]
```diff
@@ -434,6 +435,9 @@ func (k *Keeper) Liquidate(
 		totalCollateralRemain = totalCollateralRemain.Add(status.RemainCollateral)
 	}
 
+	fmt.Println("sold", sold)
```
Comment (hieuvubk) [inline x/vaults/keeper/vault.go]: can be removed

File: x/vaults/keeper/vault.go [hunk 2]
```diff
@@ -458,31 +462,32 @@ func (k *Keeper) Liquidate(
...
+		for _, vault := range liquidation.LiquidatingVaults {
+			collateralRemain := liquidation.VaultLiquidationStatus[vault.Id].RemainCollateral
+
+			if collateralRemain.Amount.Equal(math.ZeroInt()) {
+				vault.CollateralLocked.Amount = math.ZeroInt()
+				vault.Debt.Amount = math.ZeroInt()
+				vault.Status = types.CLOSED
+				continue
+			}
+			fmt.Println("current debt", vault.Debt)
```
Comment (hieuvubk) [inline x/vaults/keeper/vault.go]: ditto

File: x/vaults/types/params.go [hunk 3]
```diff
@@ -57,18 +52,12 @@ func (m Params) Validate() error {
...
 }
 
 func validateRecalculateDebtPeriod(i interface{}) error {
 	return nil
 }
-func validateLiquidatePeriod(i interface{}) error {
-	return nil
-}
 
 func validateStabilityFee(i interface{}) error {
```
Comment (hieuvubk) [inline x/vaults/types/params.go]: should move these validate to `ActiveCollateralProposal` and `UpdateCollateralProposal`

**Review by hieuvubk — APPROVED** [review]
overall lgtm, leave some nit

**Comment by hieuvubk** [comment]
@DongLieu also could u fix all the unit test. We dont want CI fail at `main`

**Comment by DongLieu** [comment]
> @DongLieu also could u fix all the unit test. We dont want CI fail at `main`

ok

**Pull request merged**